### ISSUE TITLE
The nuvolaris-documentation about development cannot be generated locally

### ISSUE THREAD

**Issue by dcremonini** [nuvolaris/nuvolaris#365]
The documentation present in the directory
`nuvolaris-documentation/modules/development/pages`
is created starting from the download of some .md (MarkDown) pages from OpenWhisk, that are converted to .adoc (AsciiDoc) pages.

This step, which is performed by the task named `download` in 
`nuvolaris-documentation/Taskfile.yml`,
cannot run in the Devcontainer provided with Nuvolaris (or at least, I cannot).

This issue prevents checking if the documentation is correctly created, and any error ends up in the target branch when a pull request is merged.

**Comment by dcremonini** [comment]
I found two reasons that explain the issue.
In the Devcontainer provided with Nuvolaris:

1. The installed _Task application_ is older than the v3.28.0. The official documentation of Task states that "As of v3.28.0, Task allows you to loop over certain values and execute a command for each." (see https://taskfile.dev/usage/#looping-over-values page content as of 13 Jul 2024).
2. The _Pandoc application_ is not installed at all.


Because of the two reasons above, the _for_ command shown in the code below (that I shortened a bit) silently fails, the _curl_ command fails, and the _pandoc_ cannot run (code taken from the `download` step, contained in the file `nuvolaris-documentation/Taskfile.yml`):
```
    - for: 
      - actions
      - ...
      - rest_api
      cmd:  |
            curl  -sL {{.SOURCES}}/{{.ITEM}}.md -o {{.ITEM}}.md
            pandoc -f markdown -w asciidoc {{.ITEM}}.md -o {{.ITEM}}.adoc
```
